activemodel/lib/active_model/dirty.rb

```diff
@@ -183,6 +183,12 @@ def attribute_changed?(attr, from: OPTION_NOT_GIVEN, to: OPTION_NOT_GIVEN) # :no
         (from == OPTION_NOT_GIVEN || from == changed_attributes[attr])
     end
 
+    # allows to use saved_change_to_attribute? in attribute_will_change! method
+    # in order to remove deprecation warning caused by attribute_changed? method
+    # in active_record/attribute_methods/dirty.rb
+    alias_method 'saved_change_to_attribute?', 'attribute_changed?'
```

Comment:
Active model should not know anything about Active Record. If we have to do something to remove this deprecation is either remove the call to `attribute_will_change!` or override it in Active Record